src/main.ts

```diff
@@ -29,11 +29,14 @@ async function run(): Promise<void> {
 
     core.info('🔮 Checking GPG private key');
     const privateKey = await openpgp.readPrivateKey(process.env.GPG_PRIVATE_KEY);
-    core.debug(`Fingerprint  : ${privateKey.fingerprint}`);
-    core.debug(`KeyID        : ${privateKey.keyID}`);
-    core.debug(`Name         : ${privateKey.name}`);
-    core.debug(`Email        : ${privateKey.email}`);
-    core.debug(`CreationTime : ${privateKey.creationTime}`);
+    core.debug(`Fingerprint     : ${privateKey.fingerprint}`);
+    core.debug(`KeyID           : ${privateKey.keyID}`);
+    core.debug(`Name            : ${privateKey.name}`);
+    core.debug(`Email           : ${privateKey.email}`);
+    core.debug(`CreationTime    : ${privateKey.creationTime}`);
+    if (process.env.FINGERPRINT) {
```

Comment:
I'm going to migrate env vars to [`inputs`](https://help.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs) soon. Can you use an input instead please? Thanks.